core/src/miner/mem_pool.rs

```diff
@@ -429,12 +429,14 @@ impl MemPool {
 
     // Recover MemPool state from db stored data
     pub fn recover_from_db<C: AccountData + BlockChainTrait>(&mut self, client: &C) {
+        let recover_block_hash = client.chain_info().best_block_hash;
+        let recover_block_id = BlockId::Hash(recover_block_hash);
         let fetch_account = |p: &Public| -> AccountDetails {
             let address = public_to_address(p);
-            let a = client.latest_regular_key_owner(&address).unwrap_or(address);
+            let a = client.regular_key_owner(&address, recover_block_id.into()).unwrap_or(address);
```

Comment:
@majecty Would you explain the difference of the latest_regular_key_owner and the regular_key_owner with the best_block_hash?

Reply:
@sgkim126 
There is a possibility that fetch_account returns different results in the mempool::add function if the best block is changed between two fetch_acccount calls.

When running the TPS test, mempool::add function is called more than 1000 per second. When running the TPS test, we found that mempool's sequence calculation logic missed sometimes. After changing fetch_account to query state with explicit best block hash, the problem is disappeared.